pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -154,16 +154,12 @@ func (b *ConsoleServerCLIConfigBuilder) Capabilities(capabilities []operatorv1.C
 
 func (b *ConsoleServerCLIConfigBuilder) AuthConfig(authnConfig *configv1.Authentication, apiServerURL string) *ConsoleServerCLIConfigBuilder {
 	switch authnConfig.Spec.Type {
-	case "", configv1.AuthenticationTypeIntegratedOAuth:
+	case "", configv1.AuthenticationTypeIntegratedOAuth, configv1.AuthenticationTypeNone:
```

Comment:
Let's add a comment explaining the change. We don't disable auth since the internal OAuth server isn't disabled even with auth type none and cluster admins can set up a custom `webhookTokenAuthenticator` that works with the integrated OAuth server tokens.